packages/components/pie-toast-provider/src/index.ts

```diff
@@ -1,20 +1,143 @@
-import { LitElement, html, unsafeCSS } from 'lit';
-import { RtlMixin, defineCustomElement } from '@justeattakeaway/pie-webc-core';
-
+import {
+    LitElement,
+    html,
+    nothing,
+    unsafeCSS,
+    type PropertyValues,
+} from 'lit';
+import { state, property } from 'lit/decorators.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import {
+    RtlMixin,
+    defineCustomElement,
+    dispatchCustomEvent,
+} from '@justeattakeaway/pie-webc-core';
+import '@justeattakeaway/pie-toast';
 import styles from './toast-provider.scss?inline';
-import { type ToastProviderProps } from './defs';
+import {
+    defaultProps,
+    PRIORITY_ORDER,
+    type ToastProviderProps,
+    type ExtendedToastProps,
+    ON_TOAST_PROVIDER_QUEUE_UPDATE_EVENT,
```

Comment:
Nit: I don't think we need `_EVENT` as that's already implied by `ON_`

Reply:
that current convention in the codebase is to postfix them with _EVENT so I would stick with it for consistency. 

<img width="1280" alt="Screenshot 2025-01-08 at 15 10 34" src="https://github.com/user-attachments/assets/952198ef-0a88-4166-a20f-1725ea3336e1" />